src/server/app/admin/controllers/users/users.password.server.controller.js

```diff
@@ -219,9 +196,13 @@ exports.reset = (req, res, next) => {
 				html: emailHTML
 			};
 
-			smtpTransport.sendMail(mailOptions, (error) => {
-				done(error, 'done');
-			});
+			emailService.sendMail(mailOptions)
+				.then((result) => {
+					done(null);
+
+				}, (error) => {
```

Comment:
Updated to reflect how this was handled in the first email when a reset is requested